src/content.ts

```diff
@@ -143,3 +144,26 @@ config.getAsync("modeindicator").then(mode => {
         if (config.get("modeindicator") !== "true") statusIndicator.remove()
     })
 })
+
+// loading style toggles on page load
+config.getAsync("styletoggles").then(styletoggles => {
```

Comment:
Is there a reason you can't use `DocStart` autocmds?

Reply:
Yes.. first loading the stylemod on page has a different logic than `style -t` (toggle mode). To use autocmd on DocStart I would have to include a hidden load mode on excmd `style`, which I was not sure about as it was already quite long. 

Also, I'd have to deal only `string.search` on DocStart to match  the filter criteria instead of just working on `url.href`, `url.host` and `url.hostname`. 

I'd be happy to hear any suggestions. 🙂